### ISSUE TITLE
Wiki cleanup

### ISSUE THREAD

**Issue by Gitoffthelawn** [uBlock-LLC/uBlock#1426]
I have cleaned up parts of the Wiki in gorhill's uBlock Origin repo.  This has included typographical corrections, technical corrections, updates to reflect functionality changes, formatting improvements, and more.  I'm not adding new information (except description of functionality changes) or changing anything that is not in error.

I've just worked on a few key sections that I thought would help others the most.

Does this repo want those changes too?  Which is the best way to bring them over?  Would it be better if I made the edits here instead and somehow make them available to gorhill?  Would it be even better if there was a separate repo for the Wiki since improvements to it will likely benefit all uBlock repos?

**Comment by chrisaljoudi** [comment]
Hi!

Yes, sure! That sounds great, and it'd be much appreciated to have them.

There is already a separate git repo for the Wiki; GitHub just doesn't expose it as a full-fledged repo.

The URL for cloning it is https://github.com/chrisaljoudi/uBlock.wiki.git

Feel free to clone it, make your changes, and then sharing the diff.

Thank you for your help, @Gitoffthelawn. :)

**Comment by Gitoffthelawn** [comment]
@chrisaljoudi You're welcome.  Thank YOU for all your hard work!  :)

I will post back here if there are any snags in the process.

**Comment by Gitoffthelawn** [comment]
LOL... first snag... I get a 404 on the link you provided! :)

**Comment by chrisaljoudi** [comment]
@Gitoffthelawn right: GitHub doesn't provide a web front-end for wikis.

`git clone https://github.com/chrisaljoudi/uBlock.wiki.git` should work. :)

**Comment by Gitoffthelawn** [comment]
@chrisaljoudi 

Thanks.  Until this thread, the git web interfaces have been sufficient for my needs (although sometimes a bit lacking).  I'll have to decide if I want to invest the time to download the executable command-line tools and learn their syntax.  Looks pretty simple though.

**Comment by chrisaljoudi** [comment]
@Gitoffthelawn Oh, I understand.

Alternatively, feel free to simply edit the wiki via the web interface. Everyone has the ability to edit.

**Comment by Gitoffthelawn** [comment]
@chrisaljoudi That's what I've been doing so far.  But I've been doing it on gorhill's fork because that's what I'm testing at the moment.  Is there a simple way to get the Wiki changes applied everywhere using the web interface (if everyone wants them!)?

**Comment by Gitoffthelawn** [comment]
I've updated some of the Wiki pages here to reflect some of the changes I made to the one on gorhill's fork.  It's a PITA for sure, since the URLs are different for many links, and different updates had been made to each Wiki prior to my updates.

I'm thinking perhaps there should be a separate github repository for the Wiki and it's images for the master and all forks.  If different forks persist, that could also provide a nice way for users (and potential users) to learn about the differences between them (if the Wiki is kept up to date with the differences).

I see both the advantages and disadvantages of having forks for the code, but I think it makes sense to unify the documentation if possible.
